src/main/java/fi/metatavu/soteapi/rss/news/NewsUpdateModel.java

```diff
@@ -11,19 +11,23 @@
   private String slug;
   private String originId;
   private String categorySlug;
+  private String created;
+  private String modified;
   private Long orderIndex;
   
   public NewsUpdateModel() {
   }
   
   public NewsUpdateModel(String title, String content, String slug, String originId, String categorySlug,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)